SpeedyTransforms/src/SpeedyTransforms.jl

```diff
@@ -25,7 +25,9 @@ import LowerTriangularArrays: lm2i, get_lm_range, get_2lm_range
 # TRANSFORM
 export  SpectralTransform,
         transform!,
-        transform
+        transform,
+	_fourier!,
```

Comment:
please don't export those

Reply:
Ah, that was just for testing, thanks for spotting it!